mutest.gemspec

```diff
@@ -39,8 +39,4 @@ Gem::Specification.new do |gem|
   gem.add_runtime_dependency('anima',         '~> 0.3.0')
   gem.add_runtime_dependency('concord',       '~> 0.1.5')
   gem.add_runtime_dependency('regexp_parser', '~> 0.4.1')
-
-  gem.add_development_dependency('rubocop-rspec', '~> 1.10.0')
```

Comment:
It is more code to add it to the gemspec and it all just ends up in the `Gemfile.lock` anyways. I just added it to the root since it isn't like we have a production group anyways. I think it makes more sense to just make them all top level gems in the Gemfile.